### DISCUSSION TITLE
Ability to disable markdown parsing on the input box

### DISCUSSION THREAD

**Discussion by PretzelVector** [open-webui/open-webui#6419] [Issues]
# Feature Request

**Is your feature request related to a problem? Please describe.**
Markdown input box enforces an opinionated version of markdown that doesn't conform to the rest of the markdown that open-webui uses (nor that we use regularly).

single asterisk -> bold -> double asterisk in message

single underscore -> underline -> single asterisk in message

**Describe the solution you'd like**
Ability to disable the markdown parsing on the input box.

**Describe alternatives you've considered**
N/A

**Additional context**
Multiple issues with markdown input make it seem like this feature is not yet mature enough to be on by default.

**Comment by tjbck** [comment]
Investigating, rich text input is still actively being worked on.

**Reply by PretzelVector** [reply]
Glad to hear it. I still would like the ability to turn it off.

**Comment by tjbck** [comment]
PR Welcome!

**Comment by silentoplayz** [comment]
Answered here - https://github.com/open-webui/open-webui/discussions/6477#discussioncomment-11067185
